master/docs/manual/cfg-statustargets.rst

```diff
@@ -1639,13 +1641,18 @@ Specifically, it follows the `Updating build status for commits <https://develop
 
 It uses the standard Python Twisted Agent to make REST requests to the stash server.
 It uses HTTP Basic AUTH.
-As a result, we recommend you use https in your base_url rather than http.
+As a result, we recommend you either connect over a secured network or use https in your base_url rather than http.
 If you use https, it requires `pyOpenSSL`.
 
-Configuration requires exactly 3 parameters:
+Configuration requires 3 parameters:
 `base_url` is the base url of the stash host, up to and optionally including the first / of the path.
 `user` is the stash user to post as
 `password` is the stash user's password
+There are also two optional parameters:
+`key_format` is a python format string used to define the aggregation key for builds in stash.
```

Comment:
I know that `python` is not spelled consistently over the documentation, however I'd prefer that new docs has `Python` all over... :smile: